From eb3a1b6732ab82e256c7271e196c557d58829ad8 Mon Sep 17 00:00:00 2001
From: Cezary Rojewski <cezary.rojewski@intel.com>
Date: Tue, 14 Aug 2018 12:41:13 +0200
Subject: [PATCH 188/292] ASoC: Intel: Add ICL machine drivers table.

Declare ICL machine drivers table within common directory as for other
Intel machine drivers.

Change-Id: I67ead17edb1bc469894a053e7d85b67409eeb2e6
Depends-On: I3c2c0ee4eeffddb495dfd765788b9cc861e7741e
Signed-off-by: Cezary Rojewski <cezary.rojewski@intel.com>
Reviewed-on:
Reviewed-by: Uzieblo, Olgierd <olgierd.uzieblo@intel.com>
---
 sound/soc/intel/skylake/skl.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/sound/soc/intel/skylake/skl.c b/sound/soc/intel/skylake/skl.c
index e6450745a4f6..9441c73abad8 100644
--- a/sound/soc/intel/skylake/skl.c
+++ b/sound/soc/intel/skylake/skl.c
@@ -1254,6 +1254,9 @@ static const struct pci_device_id skl_ids[] = {
 	{ PCI_DEVICE(0x8086, 0xa348),
 		.driver_data = (unsigned long)&snd_soc_acpi_intel_cnl_machines},
 #endif
+	/* ICL */
+	{ PCI_DEVICE(0x8086, 0x34c8),
+		.driver_data = (unsigned long)&snd_soc_acpi_intel_icl_machines},
 	{ 0, }
 };
 MODULE_DEVICE_TABLE(pci, skl_ids);
-- 
2.21.0

